src/components/Form/ExperienceRating/style.scss

```diff
@@ -0,0 +1,294 @@
+@import '~styles/colors';
```

Comment:
Do you really need all of this CSS / Styling stuff? 
I took a look at the original PR and `style.scss` file there is much more simple.

Reply:
U r right. We don't need this. Removed it in the latest commit